.changeset/six-tips-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-argo-cd-backend': major
```

Comment:
```suggestion
'@roadiehq/backstage-plugin-argo-cd-backend': minor
```
A minor version bump should be enough, what do you think? Are we introducing breaking change here?